integrations/extensions/starter-kits/language-model-conversational-search/README.md

```diff
@@ -17,7 +18,79 @@ All examples in this starter kit require that you use the [new IBM watsonx Assis
+
+- `es_model`: the ELSER model that will be used in Elasticsearch inference pipeline when processing a user query
+- `model_id`: The id of the watsonx model that you select for this action. Defaults to `meta-llama/llama-2-70b-chat`. If you keep this default, be sure to comply with the [Acceptable Use Policy for this model](https://ai.meta.com/llama/use-policy/).
+- `model_input`: The input to the watsonx model. This is set in an expression in Step 5 of the "Generate Answer" action. You MAY change that expression to do prompt engineering. If you wish to do so and are using the default model, be sure to research [guidelines for prompting Llama 2](https://www.pinecone.io/learn/llama-2/). In our experience, this combination of prompt and model is quite effective at producing high-quality answers when it has useful content and it does _often_ say that it doesn't know when it does not have useful content (as instructed in our prompt). However, it does _sometimes_ provide answers that are not supported by its evidence so consider other models, prompt expressions, or additional logic to reduce the generation of invalid answers.
+- `model_parameters_max_new_tokens` : The maximum number of new tokens to be generated. Defaults to 300.
+- `model_parameters_min_new_tokens`: The minimum number of the new tokens to be generated. Defaults to 1.
+- `model_parameters_repetition_penalty`: Represents the penalty for penalizing tokens that have already been generated or belong to the context. The range is 1.0 to 2.0 and defaults to 1.1.
+- `model_parameters_stop_sequences`: Stop sequences are one or more strings which will cause the text generation to stop if/when they are produced as part of the output. Stop sequences encountered prior to the minimum number of tokens being generated will be ignored. The list may contain up to 6 strings. Defaults to [].
+  _Note_: For some models, you may want to set this to ["\n\n"] (i.e., stop whenever the model produces a double newline) because that can be a reliable indication that the model is shifting to a new topic. However, that is generally unhelpful for chat models like the one that we're using in this kit, because those models are unlikely to suddenly shift to a new topic. The model we're using in this kit does often include double newlines within a coherent answer, so we don't recommend that setting with this model.
+- `model_parameters_temperature` : The value used to control the next token probabilities. The range is from 0 to 1.00; 0 makes it deterministic.
+- `model_response`: The text generated by the model in response to the model input.
+- `passages` : Concatenation of top search results.
+- `query_body`: The query body that will be sent to Elasticsearch search index. The query body is set and used by the Search action.
```

Comment:
The query body in step 3 of the Search action assumes that you have an ELSER field in your schema named `text_expansion` and that you want to do an ELSER search on that field only.  If you want to do an ELSER search on some other field, a non-ELSER search, or a hybrid search, you need to update the query body assignment in step 3 of the Search action.  I would mention this in the documentation and maybe provide an assortment of reasonable example values.  It should probably go in another section, because it would get lost as just extra sentences in this list.

Reply:
Good point. I added more query body examples/references for different types of queries here https://github.com/watson-developer-cloud/assistant-toolkit/pull/233/files#diff-04d57dc0d43f09b15d9d739d5b22ae2799d75c45151c56605a020ad6be22e3deR29-R51